pipeline/data_page/get_all_data_availability.py

```diff
@@ -608,12 +596,13 @@ def main(
     )
     assert omics_summary.index.is_unique
 
-    # Long Reads
+    #################
+    ### Long Reads ###
+    #################
     long_reads_summary = None
-    if depmap_long_reads_gcloud_loc is not None:
+    if len(depmap_long_reads_taiga_ids) > 0:
```

Comment:
@snwessel This should skip adding the data to the plot for the public environment. We should double check this once public is staged